staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -276,13 +273,13 @@ func (r *crdHandler) removeDeadStorage() {
 }
 
 // GetCustomResourceListerCollectionDeleter returns the ListerCollectionDeleter for
-// the given uid, or nil if one does not exist.
-func (r *crdHandler) GetCustomResourceListerCollectionDeleter(crd *apiextensions.CustomResourceDefinition) finalizer.ListerCollectionDeleter {
+// the given uid, or nil if an error occurs.
```

Comment:
the "or nil ..." is not needed here. We have the error result type now (the semantics is standard, no need to document it)